libraries/AC_PrecLand/AC_PrecLand_StateMachine.cpp

```diff
@@ -1,5 +1,9 @@
-#include "AC_PrecLand_StateMachine.h"
+#include "AC_PrecLand_config.h"
+
+#if AC_PRECLAND_STATEMACHINE_ENABLED
```

Comment:
I don't think we can have `PRECLAND` without this statemachine; not sure it deserves its own define, or whether is hould just be `AC_PRECLAND_ENABLED` here instead.

The statemachine is just an encapsulation of the inner-workings of the library, not a backend.